### PR TITLE
Adds the Objection Gavel, a lawyer-specific traitor item.

### PR DESCRIPTION
**Pull request by harmonyn**
http://objection.mrdictionary.net/go.php?n=8357838

Basically, it's timestop. All of the following numbers may be adjusted upon feedback here, as they are largely pulled out of my ass:

**TC COST:** 14
**COOLDOWN:** 5 minutes
**TIMESTOP DURATION:** 200 ticks normally, 400 ticks in the courtroom

### CODE REVIEW

**Comment by kevinz000** [comment]
20 seconds is strong to say the least.

**Comment by Arianya** [comment]
Reasonable as a starting point, IMO, since the lawyer is one of the weakest tator tots unless he can convince sec to deputize him, which is pretty rare unless sec get a false flag indication of rev or the like.

We can always nerf at a later date.

**Comment by bgobandit** [comment]
yeah, usual area, I didn't fuck with that

**Comment by XDTM** [comment]
Even the wizard's timestop isn't 20 seconds. The cooldown does balance it somewhat, but this is a free kill on anyone.

**Comment by Arianya** [comment]
If you can get within one tile of them you could have chopped off their leg with a desword that has no cooldown, so I'm not sure that's really problematic. This is 2 TC cheaper but has 5 force, a 5 minute cooldown even if you miss and is a hard tell that you're tator tot, so IMO it seems fine.

File: code/game/objects/items/weapons/courtroom.dm [hunk 1]
```diff
@@ -18,6 +18,38 @@
...
+	object(user)
+
+/obj/item/weapon/gavelhammer/objection/afterattack(atom/O, mob/user, proximity)
+	object(user)
+
+/obj/item/weapon/gavelhammer/objection/proc/object(mob/living/carbon/user)
+	if(cooldown < world.time)
+		cooldown = world.time + 3000 //5 minutes
+		var/area/A = get_area(loc)
+		if(A.name == "Courtroom" || A.name == "Holodeck - Chapel Courtroom")
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/courtroom.dm]: :(

Comment (bgobandit) [inline code/game/objects/items/weapons/courtroom.dm]: ?

**Comment by Repukan** [comment]
Too strong.

Saying Lawyer is one of the weakest traitors is an overstatement when they start with access to sec channel and brig access. Furthermore giving something a buffed wizard spell is unbalanced when crates and other means to get it exist.

**Comment by MimicFaux** [comment]
I was under the impression that all of the role-specific items were excluded from crates.
As an aside, my first thought when I saw this wasn't '20 second stun' but rather "Fuck yeah, 20 seconds to monologue evilly"

**Comment by Repukan** [comment]
That's cool and all but what you'll do vs what the playerbase will do is entirely different.

How do you even counter timestop.

**Comment by Repukan** [comment]
And I've seen people get His Grace in crates before which is why I mentioned them.

**Comment by TrustyGun** [comment]
His grace can appear in surplus crates for some reason, but not any of the other role restricted traitor items can

**Comment by Kraseo** [comment]
>How do you even counter timestop.

hahahahahah how is time & space real nigga ahhahaha just walk away nigga hahahah

**Comment by KorPhaeron** [comment]
>people upset forever because stands didn't fit the traitor theme
>now everyone is adding a wizard spell for each traitor type

hmm

**Comment by Robustin** [comment]
You counter timestop by screaming your location on radio and getting people to robust the lawyer while he tries to toolbox you to death or whatever.

**Comment by Repukan** [comment]
It's a 20 second timestop.

Not even wizards get that much duration.

You could beat the shit out of one person, and even if someone comes they would be caught in the time stop field. Which you can beat the shit out of too. Projectiles would be stopped too, meaning you can easily sidestep them.

**Comment by Robustin** [comment]
Xenobiologists have been able to do MINUTES of timestop just by spamming sepia slimes (really easy by Xenobio standards) but nobody does it because timestop is just kinda clunky.

Granted this is easier than injecting slime cores but I simply don't see this as OP given the cooldown and inherent limitations of timestop.

**Comment by KorPhaeron** [comment]
@tgstation/commit-access 

I don't think this really makes sense thematically for the lawyer to have wizard powers and I'll probably be closing this unless someone else cares more than me

**Comment by bgobandit** [comment]
I mean it's also xenobiology powers

**Comment by Robustin** [comment]
Xenobio is practically "Hogwarts in Space" though, Kor has a point.

**Comment by Repukan** [comment]
Except sepia slimes have a period before it kicks in, so it's a lot of micro and pacing if you want "minutes."

Xenobio being unbalanced doesn't mean it's good or healthy for the game to make other departments unbalanced too.

**Comment by KorPhaeron** [comment]
@bgobandit 

Whacky slime magic research in science doesn't mean that the corporate spy suddenly needs to be dio brando

**Comment by KorPhaeron** [comment]
We don't really have any broad theme across the whole game but at least the different factions can remain internally consistent lore wise

**Comment by bgobandit** [comment]
I mean I'm one of the main proponents of the "magic is weird on a space station" but we've already obliterated that line, I mean look at His Grace

**Comment by KorPhaeron** [comment]
Chaplain has always been a weird magic outlier and voice of god isn't a syndicate or station item

**Comment by Arianya** [comment]
His Grace definitely is weird and not-really-tied to the theme though.

I guess this is probably dead in the water because Design Lead veto, but it would be neat to atleast testmerge it.

Also, last time I bothered to check, flashbangs are still a counter to timestop, because they detonate even if thrown into the area of the timestop.

**Comment by KorPhaeron** [comment]
Oh yeah no I doubt it will be unbalanced, it just seems weird.

His Grace is also weird but that hit critical mass of people thinking it was a cool idea so it wasn't worth arguing against.

**Comment by Arianya** [comment]
The main issue is that I can't see it existing outside of role-specific traitor items. It wouldn't go in wizard because its just an arbitrary upgrade and Wizards have too many cool toys to combo it with, and it can't become lavaland loot because its too powerful for lavaland.

**Comment by Jordie0608** [comment]
I like the idea, and not just because of ace attorney, but I agree it's a bit strange and seemingly strong; perhaps a different effect? Area paralyse? Force drop hands? Force reaction poses? I'm not sure.

**Comment by bgobandit** [comment]
area paralyse would be cool and the same effect but also even more OP

force drop hands is mechanically cool but doesn't even go

force reaction poses would be fucking awesome but we don't have the sprites unfortunately

**Comment by PKPenguin321** [comment]
you dont need sprites just emotes

**Comment by coiax** [comment]
@bgobandit keep the long duration, just make everyone affected by the timestop GODMODE and immune to stuns, and lower the cost. This is a device for doing monologues, or escaping, not for beating someone to death with a wrench.

**Comment by Repukan** [comment]
>traitor lawyer teams up with other traitors
>They activate bombs or maxcaps and survive during the timestop

Or 

>RCD walls around the people you time stopped

**Comment by optimumtact** [comment]
Well you can use a spanner as a hammer if you really try hard enough

**Comment by coiax** [comment]
@Repukan that sounds hilarious

**Comment by coiax** [comment]
Also @Repukan I didn't say the lawyer was affected.

**Comment by Repukan** [comment]
Sure it does, but creatively enough you can bypass the no kill flag and still manage to remove people. The duration is too high, overall too strong in general.

**Comment by Repukan** [comment]
Lawyer doesn't need to be affected for the second one.

**Comment by coiax** [comment]
It's a five minute cooldown, and you can buy the far better killing weapon of an ebow for less TC.

**Comment by FantasticFwoosh** [comment]
*activates live syndicate hand bomb*

Timestop the HOS, take gun foolishly in hand & ID and replace with syndiebomb, shoot the grenade out of the hand (and or seat the HOS on a chair to obstruct immediate movement)

**Comment by coiax** [comment]
All of these convoluted methods seem hilarious, rather than points against the item.

**Comment by XDTM** [comment]
Should last about 5 seconds, with less cooldown (1-1.30 minutes), and with only ~20 seconds cooldown if used in the courtroom, instead of increasing the duration. That way you can throw objections in a court case more than once per courtroom case, and if people don't want to be vulnerable to it they can get out of the courtroom.

**Comment by coiax** [comment]
Do some lawyer RP with a traitor item, so security can ~~arrest~~ kill you immediately for valids? Top kek.

**Comment by KorPhaeron** [comment]
I slept on this and I still think letting the lawyer use magic is incredibly silly

**Comment by ExcessiveUseOfCobblestone** [comment]
Can't you just BS it with stolen chronolegion technology like you did with Stands [""""nanite tech"""""] and maybe change the effect to make it more like stolen from a chronolegionnaire and less of a direct rip from wizard ? It can be a spell ripoff in the code, but it can be thematically non-magical.

**Comment by KorPhaeron** [comment]
The syndicate being able to stop time when they're still using revolvers just seems bizarre

**Pull request closed without merging**